src/Util/iOSVersions.h

```diff
@@ -0,0 +1,9 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#include <Availability.h>
+
+#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(major, minor) \
```

Comment:
Yeah, we *could* move iOSVersions to SignalProtocolKit. I guess I was being lazy until we used it there twice.